src/main/scala/com/gu/liveappversions/S3Storage.scala

```diff
@@ -2,28 +2,35 @@ package com.gu.liveappversions
 
 import java.io.ByteArrayInputStream
 
+import com.amazonaws.AmazonServiceException
 import com.amazonaws.services.s3.AmazonS3ClientBuilder
 import com.amazonaws.services.s3.model.{ CannedAccessControlList, ObjectMetadata, PutObjectRequest, PutObjectResult }
 import com.gu.config.Aws
 import com.gu.config.Config.Env
-import com.gu.liveappversions.ios.BuildOutput
 import com.gu.liveappversions.ios.Lambda.logger
+import io.circe.Json
 import io.circe.syntax._
 
+import scala.io.Source
 import scala.util.{ Failure, Success, Try }
 
-object S3Uploader {
+object S3Storage {
 
   private val s3Client = AmazonS3ClientBuilder.standard()
     .withCredentials(Aws.credentials("developerPlayground"))
```

Comment:
I know this isn't connected to this PR, but how comes it's not in "mobile"?

Reply:
`.withCredentials(Aws.credentials("developerPlayground"))` 

This line allows us to use a bucket in the `Developer Playground` account when [running tests locally](https://github.com/guardian/live-app-versions/pull/24/files/ecae68a838a69fff9d2f882e9289ab3e325bd87c#diff-5a04af1653465858b8d53172d70f2b1eR9). Since the bucket is only used for local test data I didn't want to pollute the `Mobile` account. The 'real' bucket used by the lambdas is `static-content-dist`, which does live in the `Mobile` account.